crypto/encode_decode/decoder_pkey.c

```diff
@@ -231,10 +233,42 @@ static void collect_name(const char *name, void *arg)
     data->error_occured = 0;         /* All is good now */
 }
 
+/*
+ * The input structure check is only done on the initial decoder
+ * implementations.
+ * TODO(4.0): Investigate if an input structure condition would be feasible
```

Comment:
Create an issue please instead of creating TODO's

Reply:
So a far future TODO (feature?) issue, that we will have hanging around untouched, and that *someone* will resurrect in a few years?

Reply:
If you have to review the issues for each release then it should get looked at..
Adding more TODO's when we are trying to remove them seems counterproductive.

Reply:
I see the futility of having issues that are years old hanging around.

For the moment, I'm more tempted to simply drop that TODO and hope that someone will see the possible enhancement some time in the future...  or that I will remember it.